share/sql/common.go

```diff
@@ -33,14 +34,28 @@ func init() {
 }
 
 func getDb(c config) (*gorm.DB, error) {
+	gormCfg := &gorm.Config{
+		DisableForeignKeyConstraintWhenMigrating: false,
+	}
 	switch c.Engine {
 	case "sqlite":
```

Comment:
Our schema changes work well with sqlite? Was it tested?

Reply:
Good enough that the unit tests don't fail, and they use sqlite :)

Reply:
Maybe we should make an action out of them though, so they run automatically on PRs

Reply:
Good idea

Reply:
I think we have that already in the Reva repo, don't we?

Reply:
I think we do. Anyway shouldn't be too hard